content/kubernetes/networking/routes.md

```diff
@@ -14,7 +14,7 @@ aliases: [
 ]
 ---
 
-OpenShift routes allow requests to be routed to the REDB from outside the cluster with the need for an Ingress.
+OpenShift routes allow requests to be routed to the database from outside the cluster. For more information about routes, see [OpenShift documentation](https://docs.openshift.com/container-platform/4.13/networking/routes/route-configuration.html).
```

Comment:
it also routing to the cluster api - please consider adding to the cluster API and the databases.maybe